iwyu_test_util.py

```diff
@@ -429,8 +429,7 @@ def _GetLaunchArguments(cc_file):
   return args
 
 
-def TestIwyuOnRelativeFile(test_case, cc_file, cpp_files_to_check,
-                           verbose=False):
+def TestIwyuOnRelativeFile(cc_file, cpp_files_to_check, verbose=False):
```

Comment:
There seems to be a lot going on at once in this commit. I wonder if you could separate it into a few separate refactors to set the stage, and then one or more distinct commits to add new functionality?

Reply:
@kimgr I used `IN LISTS` syntax as @zaufi suggested, and I split the big commit into smaller atomic changes.